corona-warn-app/cwa-app-android
# [UI] Family test selection and consent screen (EXPOSUREAPP-12316, EXPOSUREAPP-12317)

### Whats new

Test type selection screen

![Screenshot 2022-03-17 at 18 12 56](https://user-images.githubusercontent.com/64849422/158854905-bb228014-7f5c-4991-b4bf-334f178a2e93.png)

##Family member consent consent

![Screenshot 2022-03-17 at 20 04 16](https://user-images.githubusercontent.com/64849422/158854930-08a42e74-2a66-4e8d-950a-1fba170b3b72.png)

### Story
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5895)
[Test type selection](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12316)
[Family member](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12317)

### How to test
1) scan any test
2) check test type selection fragment
3) press "Für andere" button
4) check family consent fragment
5) check that with empty name confirmation button is not enabled
6) type any name
7) check confirmation button is now enabled

### ToDo

- [x] Strings
- [x] New icons and illustration
- [x] Layouts
- [x] Wiring up
